styleguide/components/21-atoms/lists/_list.scss

```diff
@@ -3,16 +3,65 @@ ol {
   position: relative;
   margin-left: 1rem;
 
+  li {
+    font-family: $default-font-family;
+    font-size: .9rem;
+    line-height: 2.1;
+  }
+
   &.links {
     @extend %list-no-style;
   }
 
+  &.iconlist,
```

Comment:
You cannot use these on ordered-lists as they defeat the purpose of that element.

Reply:
Oh, so only on the <ul></ul> tag?

Reply:
I changed the code so they are only usable on ul lists.